README.md

```diff
@@ -1,2 +1,41 @@
-# join-the-team
-Information about working with the Contrast Engineering Team.
+# Join the Team
+At Contrast, our goal is the make the Internet safer day by day. We are always talking with talented and creative technologists who share this goal. We've built some amazing technology thus far and are shaking up the way the world looks at application security. We know that our products can get better with new voices and ideas.
+
+<a href="party.jpg" title="We love to have fun as a team.">
+<img src="party.jpg" alt="We love to have fun as a team."></a>
+<a href="boh.jpg" title="The view of our office from the water.">
+<img src="boh.jpg" alt="The view of our office from the water"></a>
+<a href="halloween.jpg" title="We love to have fun as a team.">
+<img src="halloween.jpg" alt="We love to have fun as a team."></a>
+<a href="boh2.jpg" title="We love to have fun as a team.">
+<img src="boh2.jpg" alt="We love to have fun as a team."></a>
+<a href="shane.jpg" title="We love to have fun as a team.">
+<img src="shane.jpg" alt="We love to have fun as a team."></a>
+
+Are you looking to make a difference? Are you a problem solver, but want to be challenged with complex and interesting problems. Do you believe the best work is the result of finding the simplest solution to complex challenges? Do you long to be an early contributor to a product and a company culture? Do you ever wish you were there in the early days of these startups everyone is talking about?
+
+If that's the case, we definitely want to [hear from you](https://www.contrastsecurity.com/careers)!
+
+# Opportunities to Join the Team
+It's rare that we post a specific job. Rather, we are always looking for talented engineers who want to come work with us. We have opportunities in the following areas:
+
+* Front-End and Mobile Client Development: Our UI is growing as our feature set expands. We are always interested in JavaScript enthusiasts and Mobile Client Development.
```

Comment:
I believe these should be two separate items or rephrased as something like "Client Side Development"